config/fam_cis_config

```diff
@@ -0,0 +1,20 @@
+#CIS Config file
+
+# List of memory servers that is associated with metadata manager.
+memsrv_ip :
+- 0:<ip_address_1>
```

Comment:
When we are adding cis config - add format as comment and keep only one entry 0:127.0.0.1